server/models/index.js

```diff
@@ -1 +1,76 @@
-index.js
\ No newline at end of file
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+mongoose.connect('mongodb://localhost/hunch');
+
+
+const UserSchema = new Schema({
+    name: {
+        type: String
+    },
+    points: {
+        type: Number
+    },
+    answer: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Answer'
+    }
+})
+
+const TopicSchema = new Schema({
+    topic: {
+        type: String
+    }
+})
+
+const GameSchema = new Schema({
+    currentTopic: { 
+        type: Schema.Types.ObjectId,
+        ref: 'Topic' },
+    round: {
+        type: Number
+    },
+    users: [UserSchema],
+    topics: [TopicSchema]
```

Comment:
This is a many-to-many relationship, but my reasoning for doing nesting over refs is that topics will never be updated once they're inserted in the database. Games will be assigned topics at the beginning of the game and that's it -- so we don't need to worry about updating in multiple places (I think???) and so the read/performance benefits would outweigh the negatives of duplication (I think???)
https://docs.mongodb.com/manual/core/data-model-design/#data-modeling-referencing